documentation/cloud-manager-guide/admin/role-based-permission/role-based-permissions.md

```diff
@@ -0,0 +1,57 @@
+---
+keywords:
+title: Role-based Permissions
+description: Role-based permissions
+---
+# Role-based Permissions
+
+The operations a user can perform and what they can view in Environment Operations Center differ based on their assigned role. Env Ops Center has three levels of user roles: Tenant Administrator, Environment Administrator, and Environment User. This guide outlines the permissions for each user role.
+
+## Tenant administrator
+
+A Tenant Administrator is granted permission to access all possible operations and views for all of the organization's environments. A Tenant Administrator can view and edit all environments and users, and can edit their own user details.
+
+Permissions include:
+
+| Env Ops Center Component | Permissions |
```

Comment:
Might make sense to turn the 3 tables throughout the doc into one as part of the intro to the doc so the reader can see at-a-glance which group has which permissions. And then link the table headers for "Tenant administrator" etc to the sections with more details.